frontend/src/pages/PipelineDetails.tsx

```diff
@@ -296,21 +296,18 @@ class PipelineDetails extends Page<{}, PipelineDetailsState> {
         );
 
         if (listVersionsResponse.pipeline_versions) {
-          latesetVersion = listVersionsResponse.pipeline_versions[0];
-          // Append version id to URL for create run (new run switcher call getPipelineVersion)
-          this.props.history.replace({
-            pathname: `/pipelines/details/${pipelineId}/version/${latesetVersion.pipeline_version_id}`,
-          });
-          return latesetVersion;
+          selectedVersion = listVersionsResponse.pipeline_versions[0];
```

Comment:
Can you add a test to verify that: ` If there are more than 1 pipeline version, the first version is being used`.  The way to verify that the first version is being used is to check the API call to getPipelineVersion is using the first version ID in this request.